<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helloClass</class>
 <widget class="QMainWindow" name="helloClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>844</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>hello</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="cfgfileGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <property name="title">
         <string>请选择日志配置文件：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="cfgFilePath">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseCfgFile">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>选择配置文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="parseCfgFile">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>解析配置文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="logGroupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>58</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>58</height>
         </size>
        </property>
        <property name="title">
         <string>请选择日志文件：</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="logFilePath">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="browseLogFile">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>选择日志文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="analysisLogFile">
             <property name="minimumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>125</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>分析日志文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="analysisProgressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="analysisResultTableWidget">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>5</number>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionExportToTxt"/>
    <addaction name="actionExportToExcel"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionImportExcel">
   <property name="text">
    <string>导入Excel文件</string>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="actionExportToTxt">
   <property name="text">
    <string>另存为txt文件</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>查找</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>关于工具</string>
   </property>
  </action>
  <action name="actionExportToExcel">
   <property name="text">
    <string>另存为Excel文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
